### PR TITLE
Add wyrm package

### PR DESCRIPTION
**Pull request by a-sevin**
* I've followed [this guide](https://docs.hunter.sh/en/latest/creating-new/create/cmake.html)
  step by step carefully. **Yes**

* I've tested this package remotely and have excluded all broken builds.
  Here is the links to the Travis/AppVeyor with status "All passed":

  * https://ci.appveyor.com/project/a-sevin/hunter/builds/23203978
  * https://travis-ci.org/a-sevin/hunter/builds/508609964

* I have submitted CI configs to https://github.com/ingenue/hunter targeting `pkg.template` branch,
  see this merged pull request https://github.com/ingenue/hunter/pull/348

---

### CODE REVIEW

File: cmake/projects/wyrm/hunter.cmake [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+        WYRM_BUILD_EXAMPLES=OFF
+        WYRM_BUILD_TESTS=OFF
+        WYRM_BUILD_DOCUMENTATION=OFF
+)
+
+hunter_pick_scheme(DEFAULT url_sha1_cmake)
+hunter_cacheable(wyrm)
+hunter_download(PACKAGE_NAME wyrm)
+
+hunter_add_package(pybind11)
```
Comment (ruslo) [inline cmake/projects/wyrm/hunter.cmake]: It should be part of the package itself.

File: docs/packages/pkg/wyrm.rst [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+  single: unsorted ; wyrm
+  single: pybind11 ; wyrm
+
+.. _pkg.wyrm:
+
+wyrm
+====
+
+-  `Official <https://gitlab.obspm.fr/greenflash/wyrm>`__
+-  `Hunterized <https://github.com/hunter-packages/wyrm>`__
```
Comment (ruslo) [inline docs/packages/pkg/wyrm.rst]: There is no such repository

File: docs/packages/pkg/wyrm.rst [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+-  `Example <https://github.com/ruslo/hunter/blob/master/examples/wyrm/CMakeLists.txt>`__
+-  Added by `Arnaud Sevin <https://github.com/a-sevin>`__ (`pr-N <https://github.com/ruslo/hunter/pull/N>`__)
```
Comment (ruslo) [inline docs/packages/pkg/wyrm.rst]: `s,N,1790,g`

File: cmake/projects/wyrm/hunter.cmake (hunk 1 above)
Comment (a-sevin) [reply cmake/projects/wyrm/hunter.cmake]: It's already in the package

**Comment by ruslo** [comment]
Testing:
- https://ci.appveyor.com/project/ingenue/hunter/builds/23221977
- https://travis-ci.org/ingenue/hunter/builds/508924280?utm_source=github_status&utm_medium=notification

**Comment by a-sevin** [comment]
update wyrm package to add pybind11 dependency

* https://travis-ci.org/a-sevin/hunter/builds/508964942
* https://ci.appveyor.com/project/a-sevin/hunter/builds/23224246

**Comment by ruslo** [comment]
Testing:
- https://ci.appveyor.com/project/ingenue/hunter/builds/23224721
- https://travis-ci.org/ingenue/hunter/builds/508972654?utm_source=github_status&utm_medium=notification

**Comment by ruslo** [comment]
Uploading:
- https://ci.appveyor.com/project/ingenue/hunter/builds/23226072
- https://travis-ci.org/ingenue/hunter/builds/508991900?utm_source=github_status&utm_medium=notification

**Comment by ruslo** [comment]
Released: https://github.com/ruslo/hunter/releases/tag/v0.23.141

**Pull request merged**